### PR TITLE
Instance Contributors Metadb derived table

### PR DESCRIPTION
**Pull request by jenmawe**
Porting over instance contributors derived table from LDP to Metadb. No issue associated with pull request

### CODE REVIEW

**Comment by jenmawe** [comment]
I just updated this file with the changes requested from other submissions. No "" around jsonb, added ordinality and made that lowercase.

File: sql_metadb/derived_tables/instance_contributors.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+SELECT 
+	ctbs.instance_id,
+	ctbs.instance_hrid,
+	ctbs.contributor_name,
+	ctbs.contributor_is_primary,
+	ctbs.contributor_type_id,
+	ctt."name" AS contributor_type_name,
+	ctbs.contributor_type_text,
+	ctbs.contributor_name_type_id,
+	cntt."name" AS contributor_name_type_name,
```
Comment (adoerrer) [inline sql_metadb/derived_tables/instance_contributors.sql:30]: I assume Dbeaver had sneaked in the quotes on name as well.

**Review by adoerrer — CHANGES_REQUESTED** [review]
I appologize I thought I've finished my review already...
The query works great, I've found only some smaller issues and it is missing in  the runlist, thank you.

**Review by adoerrer — APPROVED** [review]
Look and works great, thanks!

**Review by amzoss — APPROVED** [review]
Looks great!

**Comment by amzoss** [comment]
- [x] Query runs
- [x] Data output makes sense
- [x] Data types look correct
- [x] Query logic seems correct
- [x] Column names make sense and are spelled correctly
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)
- For derived tables:
    - [x] query has indexes on columns
    - [x] vacuum analyze has been added to the bottom
    - [x] table appears in runlist.txt at an appropriate point

File: sql_metadb/derived_tables/instance_contributors.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		i.id AS instance_id,
+		jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,
+		jsonb_extract_path_text(ctb.jsonb, 'name') AS contributor_name,
+		jsonb_extract_path_text(ctb.jsonb, 'primary')::boolean AS contributor_is_primary,
+		jsonb_extract_path_text(ctb.jsonb, 'contributorTypeId')::uuid AS contributor_type_id,
+		jsonb_extract_path_text(ctb.jsonb, 'contributorTypeText') AS contributor_type_text,
+		jsonb_extract_path_text(ctb.jsonb, 'contributorNameTypeId')::uuid AS contributor_name_type_id,
+    	ctb.ordinality AS contributor_ordinality
+	FROM 
+		folio_inventory.instance i
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/instance_contributors.sql:18]: We are using `AS` here even though Postgres does not require it.  I will add this.

**Pull request merged**